src/modules/database/dto/answer.dto.ts

```diff
@@ -0,0 +1,25 @@
+import { v4 as uuidv4 } from 'uuid';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class AnswerDto {
+    @ApiProperty({ description: 'answer id', example: uuidv4() })
+    id?:string;
+
+    @ApiProperty({ description: 'title text', example: 'title for answer' })
+    titleText: string;
+
+    @ApiProperty({ description: 'description text', example: 'description for answer' })
+    descriptionText: string;
+
+    @ApiProperty({ description: 'main text', example: 'main text in answer' })
+    text: string;
```

Comment:
if you add an example, enter an example, now it looks like a field description

Reply:
done